src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngineUnitTest.java

```diff
@@ -0,0 +1,260 @@
+
+        final int alt = 5;
+        final int ref = 3;
+
+        final SimpleInterval loc= new SimpleInterval("1", 69428, 69428);
+        final VariantContext vcRS = new FeatureDataSource<>(file, new VCFCodec(), null, 0).query(loc).next();
+
+        final Allele refAllele = vcRS.getReference();
+        final Allele altAllele = vcRS.getAlternateAllele(0);
+
+        final VariantContext vcToAnnotate = makeVC(refAllele, altAllele, loc);
+        final Map<String, PerReadAlleleLikelihoodMap> perReadAlleleLikelihoodMap = makeReadMap(ref, alt, refAllele, altAllele, loc.getContig(), loc.getStart() - 5);
+
+        final FeatureContext featureContext= when(mock(FeatureContext.class).getValues(dbSNPBinding, loc.getStart())).thenReturn(Arrays.<VariantContext>asList(vcRS)).getMock();
+        final VariantContext resultVC = vae.annotateContext(featureContext, null, vcToAnnotate, perReadAlleleLikelihoodMap, a->true);
+        Assert.assertEquals(resultVC.getCommonInfo().getAttribute(VCFConstants.DEPTH_KEY), String.valueOf(ref+alt));
+        Assert.assertEquals(resultVC.getID(), vcRS.getID());
+    }
+
+    @Test
+    public void testCoverageAnnotationViaEngine() throws Exception {
+        final File file= new File(publicTestDir + "Homo_sapiens_assembly19.dbsnp135.chr1_1M.exome_intervals.vcf");
+        final FeatureInput<VariantContext> dbSNPBinding = new FeatureInput<>("dbsnp", Collections.emptyMap(), file);
+
+        final List<String> annotationGroupsToUse= Collections.emptyList();
+        final List<String> annotationsToUse = Arrays.asList(Coverage.class.getSimpleName(),
+                                                            DepthPerAlleleBySample.class.getSimpleName(),
+                                                            SampleList.class.getSimpleName());
+        final List<String> annotationsToExclude= Collections.emptyList();
+        final VariantAnnotatorEngine vae = VariantAnnotatorEngine.ofSelectedMinusExcluded(annotationGroupsToUse, annotationsToUse, annotationsToExclude, dbSNPBinding);
+
+        final int alt = 5;
+        final int ref = 3;
+        final Allele refAllele = Allele.create("A", true);
+        final Allele altAllele = Allele.create("T");
+
+        final Map<String, PerReadAlleleLikelihoodMap> perReadAlleleLikelihoodMap = makeReadMap(ref, alt, refAllele, altAllele);
+        final VariantContext resultVC = vae.annotateContext(new FeatureContext(), null, makeVC(refAllele, altAllele), perReadAlleleLikelihoodMap,
+                                            ann -> ann instanceof Coverage || ann instanceof DepthPerAlleleBySample);
+        Assert.assertEquals(resultVC.getCommonInfo().getAttribute(VCFConstants.DEPTH_KEY), String.valueOf(ref+alt));
+
+        Assert.assertEquals(resultVC.getGenotype(0).getAD(), new int[]{ref, alt});
+
+        //skipped because it's not active region annotation
+        Assert.assertFalse(resultVC.getCommonInfo().hasAttribute(GATKVCFConstants.SAMPLE_LIST_KEY));
```

Comment:
Active region annotation? You don't have an active region annotation group in this branch. What exactly is this test case supposed to be testing?